### DISCUSSION TITLE
<feat> Keyboard shortcut key for <open/close sidebar>

### DISCUSSION THREAD

**Discussion by Devanshukoli** [dailydotdev/daily#872] [Feature Requests]
**TL;DR** : Add **shortcut key** for opening and closing sidebar.

Since I am started working on the bigger screen. I have found this issue OR you can say feature to be made out of it.

In the browser, there is a sidebar for various interactions to do with. and you can close it via the toggle button, right?
- so, What I am suggesting is this: Why don't add a **shortcut key** for that?

It will be a lot easier for showing the blogs. And if I have to do any kind of interaction then I have to just press that shortcut key.

What do @daily.dev thinks about it?

**Comment by idoshamun** [comment]
Hi @Devanshukoli,

Thanks a lot for the suggestion :)
